applications/luci-app-sshtunnel/htdocs/luci-static/resources/view/sshtunnel.js

```diff
@@ -0,0 +1,403 @@
+
+		o = s.taboption("general", form.ListValue, "IdentityFile", _("Key file"),
+			_("Private key file with authentication identity. The default is /root/.ssh/id_ed25519, id_rsa, id_ecdsa or id_dropbear.")
+		);
+		o.value("", "");
+		Object.keys(sshKeys).forEach(function (keyName) {
+			o.value("/root/.ssh/" + keyName, keyName);
+		});
+		o.optional = true;
+
+		o = s.taboption("advanced", form.ListValue, "CheckHostIP", _("Check host IP"),
+			_("CheckHostIP If enabled the ssh will check the host IP address in the known_hosts file. " +
+				"This allows ssh to detect if a host key changed due to DNS spoofing.")
+		);
+		o.value("yes", _("Yes"));
+		o.value("no", _("No"));
+		o.default = "no";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.ListValue, "VerifyHostKeyDNS", _("Verify Host Key DNS"),
+			_("VerifyHostKeyDNS Verify the remote key using DNS and SSHFP resource records.")
+		);
+		o.value("yes", _("Yes"));
+		o.value("no", _("No"));
+		o.default = "no";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.ListValue, "Compression", _("Use compression"),
+			_("Compression may be useful on slow connections.")
+		);
+		o.value("yes", _("Yes"));
+		o.value("no", _("No"));
+		o.default = "no";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.Value, "retrydelay", _("Retry delay"),
+			_("Delay after a connection failure before trying to reconnect..")
+		);
+		o.placeholder = 10;
+		o.default = 10;
+		o.datatype = "uinteger";
+		o.optional = true;
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.Value, "ServerAliveCountMax", _("Server Alive Count Max"),
+			_("ServerAliveCountMax The number of server alive messages which may be sent before ssh disconnect from the server.")
+		);
+		o.placeholder = 3;
+		o.datatype = "uinteger";
+		o.optional = true;
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.Value, "ServerAliveInterval", _("Server Alive Count Max"),
+			_("ServerAliveInterval Timeout interval in seconds for keep alive.")
+		);
+		o.optional = true;
+		o.placeholder = 0;
+		o.datatype = "uinteger";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.ListValue, "StrictHostKeyChecking", _("Strict host key checking"),
+			_("StrictHostKeyChecking Refuse to connect to hosts if it's key has changed."));
+		o.value("accept-new", _("Accept new and check if not changed"));
+		o.value("yes", _("Yes"));
+		o.value("no", _("No"));
+		o.default = "accept-new";
+		o.modalonly = true;
+
+		o = s.taboption("advanced", form.ListValue, "LogLevel", _("Log level"), "");
+		o.value("QUIET", "QUIET");
+		o.value("FATAL", "FATAL");
+		o.value("ERROR", "ERROR");
+		o.value("INFO", "INFO");
+		o.value("VERBOSE", "VERBOSE");
+		o.value("DEBUG", "DEBUG");
+		o.default = "INFO";
+		o.modalonly = true;
+
+
+		s = m.section(form.GridSection, "tunnelR", _("Remote Tunnels"),
+			_("When the connection initiated at RemoteAddress:RemotePort and then forwarded to LocalAddress:LocalPort")
```

Comment:
Your wording is unclear. 
Try:

`_("Forward a port or ports on remote systems to the local system.")`

Reply:
I gave proposals for those already.